serialization/src/main/scala/sbt/serialization/SerializedValue.scala

```diff
@@ -22,116 +24,107 @@ import sbt.serialization.json.{
 sealed trait SerializedValue {
   def parse[T](implicit unpickler: Unpickler[T]): Try[T]
 
-  // TODO - expose toJson, toBinary etc. (anyhting we need).
+  def toJsonString: String
+
+  // TODO toBinary if/when we add binary pickling
+
+  // private because we don't have our chosen
+  // json AST library in the public API
+  private def toJValue: JValue =
+    this match {
```

Comment:
since JsonValue is private the compiler complained about that even though the sealed trait should mean it's ok. compiler isn't that smart.